client.go

```diff
@@ -325,48 +326,71 @@ func (c *Client) Close() error {
 
 // ConsumerFunc is passed into Consumer, and is used as a callback for incoming
 // Events. It should return true if the event was processed successfully and
-// false otherwise. If ConsumerUnsafe is being used the return is ignored
+// false otherwise.
 type ConsumerFunc func(Event) bool
```

Comment:
Can this somehow accept `context, event` so the context expires when the expireseconds is reached?